packages/react-components/src/Button/Button.tsx

```diff
@@ -132,4 +133,23 @@ export default React.memo(styled(Button)(({ theme }: ThemeProps) => `
   &.isDisabled {
     color: #bcbbba;
   }
+  
+  .ui--CreateAccount-new-create-modal && {
+    box-shadow: none;
+    line-height: 1.5rem;
+
+    &.isDisabled {
+      color: #fff;
+      background: #C2C2C2;
+      box-shadow: none;  
+    }
+
+    &:not(.hasLabel) {
+      line-height: initial;
+    }
+
+    &.hasLabel {
+      padding: 0.65rem 1.1rem 0.65rem ${1.1 - ICON_PADDING}rem;
+    }
```

Comment:
This should really not know about anything in `apps-*` - specific overrides should go in there.